packages/tenant-process/src/services/tenantService.ts

```diff
@@ -66,11 +65,10 @@ const retrieveTenant = async (
 
 export async function retrieveAttribute(
   attributeId: AttributeId,
-  readModelService: ReadModelService,
-  kind: AttributeKind
+  readModelService: ReadModelService
 ): Promise<Attribute> {
   const attribute = await readModelService.getAttributeById(attributeId);
-  if (!attribute || attribute.kind !== kind) {
+  if (attribute === undefined) {
```

Comment:
```suggestion
  if (!attribute) {
```

Reply:
I put the `=== undefined` condition to make it equal to the `retriveTenant`

Reply:
In these cases I think that `!attribute` is better because it will be `true` even with falsy values (`NaN`, `null`, ...), what do you think @Carminepo2 ?